apps/ex_nvr_web/lib/ex_nvr_web/live/device_list_live.ex

```diff
@@ -97,6 +97,15 @@ defmodule ExNVRWeb.DeviceListLive do
                     Stop recording
                   </.link>
                 </li>
+                <li>
+                  <.link
+                    :if={Device.recording?(device)}
+                    href={~p"/webrtc/#{device.id}"}
+                    class="block px-4 py-2 hover:bg-gray-100 dark:hover:bg-gray-600 dark:hover:text-white"
```

Comment:
Delete this, we don't need a link on the devices.
Add a new section in `README` to add an explanation about this endpoint.